internal/controller/postgrescluster/controller_test.go

```diff
@@ -114,6 +115,9 @@ var _ = Describe("PostgresCluster Reconciler", func() {
 		test.Namespace.Name = "postgres-operator-test-" + rand.String(6)
 		Expect(suite.Client.Create(ctx, test.Namespace)).To(Succeed())
 
+		// Initialize the feature gate
+		Expect(util.AddAndSetFeatureGates("")).To(Succeed())
```

Comment:
I don't think this new check will be hard to move later but I think we are trying to remove this type of test (ginkgo) :memo: